frontend/src/component/admin/auth/PasswordAuth/PasswordAuthDialog.tsx

```diff
@@ -0,0 +1,53 @@
+import { Alert, Typography } from '@mui/material';
+import { Dialogue } from 'component/common/Dialogue/Dialogue';
+import { IAdminCount } from 'hooks/api/getters/useAdminCount/useAdminCount';
+import { IApiToken } from 'hooks/api/getters/useApiTokens/useApiTokens';
+
+interface IPasswordAuthDialogProps {
+    open: boolean;
+    setOpen: (open: boolean) => void;
+    onClick: () => void;
+    adminCount: IAdminCount;
+    tokens: IApiToken[];
+}
+
+export const PasswordAuthDialog = ({
+    open,
+    setOpen,
+    onClick,
+    adminCount,
+    tokens,
+}: IPasswordAuthDialogProps) => (
+    <Dialogue
+        open={open}
+        onClose={() => {
+            setOpen(false);
+        }}
+        onClick={onClick}
+        title="Disable password based login?"
+        primaryButtonText="Disable password based login"
+        secondaryButtonText="Cancel"
+    >
+        <Alert severity="warning">
```

Comment:
Am I reading this right if I say, we're not actually stopping people from locking themselves out and trusting that they won't click silly buttons?

Reply:
Correct, we still trust our users to do the correct thing here, we're just helping them making a more informed decision.

As per the PR description:
>This felt better than trying to disable the toggle, by still allowing the end users to make the decision, except now it should be a properly informed decision with confirmation.

It also doesn't feel trivial to make a condition that completely stops people from proceeding because it depends entirely on their use case. Maybe they want their instance to be managed by an admin SA and keep an admin API token secured just in case 🤷

Reply:
Yeah fair enough, seems like an reasonable tradeoff. Someone will absolutely lock themselves out but at least at that point we can cry on their shoulder and ask them why they would do that